File: PHP/card.php
<?php
require_once("./payment.php");
class Card extends Payment
{
    public int $number;
    public int $cvv;
    public string $date;

    public function __construct($id, $number, $cvv, $date)
    {
        parent::__construct($id);
        $this->number = $number;
        $this->number = $cvv;
        $this->date = $date;
    }
}

File: JAVA/Card.java
package JAVA;

public class Card extends Payment {
    Integer number;
    Integer cvv;
    String date;

    Card(Integer id, Integer number, Integer cvv, String date) {
        super(id);
        this.number = number;
        this.cvv = cvv;
        this.date = date;
    }
}
